core/src/test/scala/integration/kafka/api/SaslGssapiSslEndToEndAuthorizationTest.scala

```diff
@@ -24,10 +24,10 @@ import scala.collection.immutable.List
 class SaslGssapiSslEndToEndAuthorizationTest extends SaslEndToEndAuthorizationTest {
   override val clientPrincipal = JaasTestUtils.KafkaClientPrincipalUnqualifiedName
   override val kafkaPrincipal = JaasTestUtils.KafkaServerPrincipalUnqualifiedName
-  
```

Comment:
nit: removing this whitespace diff and the one below would remove this file from the PR